lib/src/main/java/io/ably/lib/transport/Defaults.java

```diff
@@ -11,15 +11,15 @@
     /* versions */
     public static final float ABLY_VERSION_NUMBER   = 1.2f;
     public static final String ABLY_VERSION         = new DecimalFormat("0.0", new DecimalFormatSymbols(Locale.ENGLISH)).format(ABLY_VERSION_NUMBER);
-    public static final String ABLY_LIB_VERSION     = String.format("%s-%s", BuildConfig.LIBRARY_NAME, BuildConfig.VERSION);
+    public static final String ABLY_AGENT_VERSION   = String.format("%s/%s", "ably-java", BuildConfig.VERSION);
```

Comment:
Since the format agent name should always be `ably-java` I've decided to use a hardcoded string, is it okay or should I create some constant for that? :thinking:

Reply:
Looks good to me.